velox/docs/monthly-updates/january-2023.rst

```diff
@@ -0,0 +1,83 @@
+********************
+January 2023 Update
+********************
+
+Core Library
+============
+
+* Added generic writer which supports generic outputs in the simple function interface and thus unblocks authoring
+ a large set of functions that operate on any type such as :func:`subscript`, :func:`array_concat`, :func:`array_intersect`.
+* Reject null aware right semi project joins and add support for null-aware left semi join project with filter.
+* Various memory pool fixes : to always return valid buffer, memory usage count leak with async data cache failures.
+* Make memory usage tracker thread safe.
+
+
+Presto Functions
+================
+
+* Add :func:`array_frequency`, :func:`json_parse`, :func:`shuffle`, :func:`sha1`,
+  :func:`spooky_hash_v2_32`, :func:`spooky_hash_v2_64`, :func:`repeat`, :func:`week`,
+  :func:`week_of_year`, :func:`json_format` scalar functions.
+* Fix error handling in :func:`least`, :func:`greatest`, :func:`like`, :func:`array_sum`, :func:`array_position`,
+:func:`regex_extract`, :func:`regex_like`, :func:`regex_extract_all`, :func:`like`,
+ :func:`makeWidthBucketArray` scalar functions.
+* Fix errors in :func:`array_except`, :func:`intersect`, :func:`date_parse`,
+    :func:`bitwise_left_shift`, :func:`bitwise_right_shift` scalar functions.
+* Fix HLL index bit length estimation.
+* Fix crash when deserializing small KLL sketch and also make them backwards compatible.
+
+Spark Functions
+===============
+
+* Add support for DATE inputs for least and greatest functions.
+* Fix year functions to match Spark's semantics.
+
+Hive Connector
+==============================
+
+* Enable installSigPipeHandler option in S3FileSystem.
+* Add support for Map columns in Parquet reader.
+* Fix null filter pushdown for ARRAY and MAP.
+
+
+Substrait
+=========
+
+* Update Substrait to 0.22.0.
+
+
+Performance and Correctness
+===========================
+
+* Fix multiple bugs in expression evaluation with TRY.
+* Fix unnecessary copy in the native Parquet reader.
+* `applyFunctionWithPeeling` should flatten single arg functions input always.
+* Revamp function selection in fuzzer.
```

Comment:
How about? : Modify expression fuzzer to randomly choose functions from a list of eligible function names instead of from a list of function signatures.